ruby.rb

```diff
@@ -0,0 +1,9 @@
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.